pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -441,6 +426,7 @@ rec {
         gtest
         icu
         libcap
+        libnbd
```

Comment:
In case anyone's wondering, this is a new upstream dependency from [squid: librbd/migration: add external clusters support + NBD stream](https://redirect.github.com/ceph/ceph/pull/63406).